src/compile/inference/file_level_definitions.pegjs

```diff
@@ -143,6 +144,18 @@ EnumDef = ENUM __ name: Identifier __ body: EnumDefBody {
     } as FLEnumDefinition;
 }
 
+// ==== Event
+
+EventArgs = LPAREN __ ParenSoup __ RPAREN { return text(); }
+EventDef = EVENT __ name: Identifier __ args: EventArgs __ SEMICOLON {
```

Comment:
Small nit: The event definition can optionally have an `anonymous` keyword after the arguments. (See [here](https://docs.soliditylang.org/en/latest/grammar.html#a4.SolidityParser.eventDefinition) for details)

Reply:
Addressed in 8fdc8af